scripts/delete.sh

```diff
@@ -131,9 +131,15 @@ if [[ "$crb_ns" == "$namespace" ]]; then
   echo "deleting cluster scope RBAC"
   kustomize build rbac/cluster-scope | kubectl delete -f - --ignore-not-found=true
 fi
+
+# Determine whether deployment is namespace-scoped before deleting runtime resources
+is_namespace_scoped=$(kubectl exec deploy/modelmesh-controller -- printenv NAMESPACE_SCOPE 2> /dev/null || echo "false") || :
 kustomize build default | kubectl delete -f - --ignore-not-found=true
 kustomize build rbac/namespace-scope | kubectl delete -f - --ignore-not-found=true
-kustomize build runtimes ${kustomize_load_restrictor_arg} | kubectl delete -f - --ignore-not-found=true
+if [[ ! "$is_namespace_scoped" == "true" ]]; then
+  echo false
```

Comment:
`echo false`?

this looks like a dev/test artifact or copy paste error? :-)

should be:

```Bash
   kustomize build runtimes ${kustomize_load_restrictor_arg} | kubectl delete -f - --ignore-not-found=true
```